utils/unittest/unittest_test.go

```diff
@@ -0,0 +1,50 @@
+package unittest
+
+import (
+	"fmt"
+	"testing"
+)
+import "os"
+
+// TestCrashTest_ErrorMessage tests that CrashTest() can check a function that crashed without any messages.
+func TestCrashTest_NoMessage(t *testing.T) {
+	f := func(t *testing.T) {
+		Crash_NoMessage()
+	}
+
+	CrashTest(t, f, "", "TestCrashTest_NoMessage")
+}
+
+// TestCrashTest_ErrorMessage tests that CrashTest() can read standard messages from stdout before a crash.
+func TestCrashTest_ErrorMessage(t *testing.T) {
```

Comment:
Updated to make helper functions lowercase https://github.com/onflow/flow-go/pull/2811/commits/23d9fdf90f5c059448af7a582ca2f45171b1009b

I kept the test function as is because the `_` helps to describe how we're testing the same function in different ways. In this case, we're testing `CrashTest` in 3 different ways: `_NoMessage`, `_ErrorMessage` and `_Logger`.